Content.Server/NPC/Systems/NPCUtilitySystem.cs

```diff
@@ -499,6 +499,26 @@ private void Filter(NPCBlackboard blackboard, HashSet<EntityUid> entities, Utili
 
                 break;
             }
+            case RemoveAnchoredFilter:
+            {
+                _entityList.Clear();
+
+                foreach (var ent in entities)
+                {
+                    if (!TryComp<TransformComponent>(ent, out var xform))
```

Comment:
use TransformQuery

Reply:
Should I use the TryComp overload that translates to 
`return EntityManager.TransformQuery.TryGetComponent(uid, out comp);`
or EntityManager.TransformQuery or the one that this NPCUtilitySystem initializes?
I think the right answer would be the overload, but I'm not sure since EntityQueries are something I haven't worked with yet.

Reply:
you can use `EntityManager.TransformQuery.TryComp` or `TryComp(ent, out TransformComponent? xform)` (cant use var for it to use the overload, see EntitySystem.Proxy.cs)

no system should really have its own transform query since entitymanager has one

Reply:
I guess that I'll use the `TryComp`. Also did a bit of blaming, it looks like this system has it's own `Transformquery` because this is older than the publication of `EntityManager.TransformQuery`